cmd/degitx-gitaly/main.go

```diff
@@ -0,0 +1,74 @@
+// MIT License. Copyright (c) 2020 CQFN
+// https://github.com/cqfn/degitx/blob/master/LICENSE
+
+package main
+
+import (
+	"log"
+	"os"
+
+	"cqfn.org/degitx/config"
+	frontend "cqfn.org/degitx/front-end"
+	"cqfn.org/degitx/front-end/gitaly/server"
+	"cqfn.org/degitx/logging"
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := cli.App{
+		Name:        "Gitaly-like front-end",
+		Usage:       "Manage degitx front-end",
+		Description: "DeGitX front-end CLI",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "config",
+				Aliases:  []string{"c"},
+				Usage:    "configuration file path",
+				Required: true,
+			},
+		},
+		Commands: []*cli.Command{
+			{
+				Name:    "run",
+				Aliases: []string{"r"},
+				Usage:   "run front-end",
+				Action:  cmdRun,
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:     "gitaly-host",
```

Comment:
@stepanvalyavskiy I think there are no reasons for such explicit argument names, we can simplify this to `host`